### DISCUSSION TITLE
renovate not running despite enabled

### DISCUSSION THREAD

**Discussion by fabricesemti80** [renovatebot/renovate#18012] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

When I cloned the repository, it used to work (https://github.com/fabricesemti80/home-gitops-cluster-old/issues/1 ) but it stopped after the initial run.

I have now recreated my repository (https://github.com/fabricesemti80/home-gitops-cluster - the new repo), and despite I have the config for renovate (https://github.com/fabricesemti80/home-gitops-cluster/blob/main/.github/renovate.json5) in place, it never tiggered on this one.

- I tried to uninstall the renovate app and then reinstall it on my account
- I also tried to set the scope to all repository / just this one repository to no avail

As this is not self-hosted solution, I do not really know, where would I see logs or what is the reason behind it is not running.
Could you please point me to the right direction...?

Thanks,
F

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment]
Hi, you can access the logs at app.renovatebot.com

It looks like you have some failing Docker images causing jobs to abort. You will need to add credentials for them or add them to the ignoreDeps list.

**Comment by fabricesemti80** [comment]
Thanks very much @rarkins .
